WireUI/Sources/WireSettingsUI/Account/BackupImportExport/Import/ImportBackupViewModel.swift

```diff
@@ -208,9 +208,10 @@ final class ImportBackupViewModel: ObservableObject {
             false
         }
 
-        let isAlertPresented = if case .success = state {
+        let isAlertPresented = switch state {
+        case .success, .restoreFailed:
             true
-        } else {
+        default:
```

Comment:
question: is it related to this PR?

Reply:
No, it also belongs to the unrelated fix mentioned.

Reply:
Moved it to another PR: https://github.com/wireapp/wire-ios/pull/2709